py/desispec/workflow/desi_proc_funcs.py

```diff
@@ -595,6 +605,9 @@ def create_desi_proc_batch_script(night, exp, cameras, jobdesc, queue, runtime=N
             fx.write("export OMP_NUM_THREADS={}\n".format(threads_per_core))
         else:
             fx.write("export OMP_NUM_THREADS=1\n")
+        if system_name == 'perlmutter-gpu':
+            fx.write("export MPICH_GPU_SUPPORT_ENABLED=1\n")
+            fx.write("module load cudatoolkit\n\n")
```

Comment:
Is there any risk if cudatoolkit was already loaded by the environment?  For everything else we require that the environment is properly configured before launching the job instead of having the job load additional modules.

Reply:
Thanks for catching this. There is no risk, but it is better to be consistent and have the necessary modules loaded by the environment rather than in the Slurm script. I've removed the loading of the module, created a [desiconda issue](https://github.com/desihub/desiconda/issues/51) about this for future installations, and modified `20220119-2.0.1/modulefiles/desiconda/20220119-2.0.1` by hand on Perlmutter to load the cudatoolkit.